lago/virt.py

```diff
@@ -166,6 +167,32 @@ def virt_path(self, *args):
     def bootstrap(self):
         utils.invoke_in_parallel(lambda vm: vm.bootstrap(), self._vms.values())
 
+    def export_entities(self, entities_names, standalone, dst_dir, compress):
+        if not entities_names:
+            entities_names = self._vms.keys()
+
+        running_ents = []
+        entities = []
+        for name in entities_names:
+            try:
+                ent = self._vms[name]
+                entities.append(ent)
+                if ent.defined():
+                    running_ents.append(ent)
+            except KeyError:
+                LOGGER.error('Entity {} does not exist'.format(name))
+                sys.exit(1)
+
+        if running_ents:
+            for ent in running_ents:
+                LOGGER.error('{} must be off'.format(ent.name()))
+            sys.exit(1)
+        # TODO: run the export task in parallel
```

Comment:
Not something that I can think on for this moment, but let keep things as simple as we can for now, and improve later.